tests/functional/views/test_scans.py

```diff
@@ -117,6 +117,8 @@ def test_scan(self):
         5. GET /reports/history
         6. GET /reports/status
         7. GET /reports/issues
+        8. POST /issue/tagFalsePositive
+        9. POST /issue/tagIgnored
```

Comment:
The standard for REST APIs is to use a dash, not camel case.

In this case, we may want to have something like:
GET /issue/<id>  <-- gets the status { status: { "ignored": false, "false-positive": true } }

POST /issue/<id> <-- sets the status

DELETE /issue/<id> <-- removes the ID